shopify/one-time-charge-template/web/routes/billing.tsx

```diff
@@ -63,7 +64,7 @@ export default () => {
                   setDisabled(true);
```

Comment:
`res` type error, might be an autocomponent issue?
![CleanShot 2024-11-06 at 10 48 36@2x](https://github.com/user-attachments/assets/2d5c4892-ee07-4381-a2b0-3d884bf5337c)

Reply:
Yeah, I think that it's an autocomponent issue